udaykalra/portfolio/target/portfolio-1/style.css

```diff
@@ -1,9 +1,144 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.bottomRow
+{
+	padding-top: 30px;
 }
-
-#greeting-container {
-  margin-top: 20px;
+.center
+{
+	margin: auto;
+	text-align: center;
 }
+.hero-image
+{
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+ url('hero.jpg');
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	height: 50%;
+	position: relative;
+}
+.hero-text
```

Comment:
Style prescribes separating rules with new lines:
https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation

Also applies to other rules in this file.

Reply:
The target directory has artifacts generated by maven, not sure how to keep them out of pull requests.